client/my-sites/email/email-providers-stacked-comparison/billing-selector/billing-selector.tsx

```diff
@@ -0,0 +1,40 @@
+import { useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent } from 'react';
+import SegmentedControl from 'calypso/components/segmented-control';
+import { TermLength } from '../provider-cards/utils';
+
+import './style.scss';
+
+interface BillingSelectorProps {
+	termLength: TermLength;
+	onTermTypeChange: ( term: TermLength ) => void;
+}
+
+export const BillingSelector: FunctionComponent< BillingSelectorProps > = ( props ) => {
+	// eslint-disable-next-line @typescript-eslint/no-empty-function
+	const { onTermTypeChange = () => {}, termLength } = props;
+	const translate = useTranslate();
+	const onTermClick = ( termLength: TermLength ) => {
+		return () => onTermTypeChange( termLength );
+	};
+
+	return (
+		<div className="billing-selector__wrapper">
+			<SegmentedControl compact className={ 'billing-selector__wrapper' } primary={ true }>
```

Comment:
Was it intentional to have both the parent div and this control have the same `className`?

(Also this could be shortened a bit as follows:)

```suggestion
			<SegmentedControl compact className="billing-selector__wrapper" primary>
```